src/actions/instances.ts

```diff
@@ -291,6 +290,44 @@ export const restoreDefaultParameterMetaOnRemote = (_instance: InstanceStateReco
 		oscQueryBridge.sendPacket(writePacket(message));
 	};
 
+export const activateParameterMIDIMappingFocusOnRemote = (instance: InstanceStateRecord, param: ParameterRecord): AppThunk =>
+	(dispatch) => {
+
+		// To activate Parameter Mapping Focus we just "touch" the normalized
+		const message = {
+			address: `${param.path}/normalized`,
+			args: [
+				{ type: "f", value: param.normalizedValue }
+			]
+		};
+
+		oscQueryBridge.sendPacket(writePacket(message));
+		dispatch(setInstance(instance.setParameterWaitingForMidiMapping(param.id)));
+	};
```

Comment:
we don't actually need to send a packet, we could just do 
`dispatch(setInstance(instance.setParameterWaitingForMidiMapping(param.id)));` without the OSC